web-app/django/VIM/apps/instruments/views/instrument_list.py

```diff
@@ -5,15 +5,17 @@
 import requests
 from django.conf import settings
 from django.core.paginator import Paginator, Page
-from django.db.models import Prefetch, QuerySet
-from django.views.generic import ListView
+from django.views.generic import TemplateView
 
-from VIM.apps.instruments.models import Instrument, InstrumentName, Language
+from VIM.apps.instruments.models import Language
 
 logger = logging.getLogger(__name__)
 
 # Constants
-SOLR_SEARCH_MARKER = "SOLR_SEARCH"
+DEFAULT_PAGE_SIZE = 20
```

Comment:
If you anticpaye using Solr in other places, then these shouldn’t in your settings file and be imported.